amp-camp/src/server/server.js

```diff
@@ -50,19 +57,48 @@ const listener = app.listen(port, () => {
 
 /** HANDLERS FOR STATIC PAGES AND STATIC FILES **/
 
+// Optimize and cache HTML responses
```

Comment:
I know our server code needs refactoring desperately. I'd hoped to do that last week. Clearly haven't gotten to do so :)

Can you leave a comment that explains in some detail how the optimization works? This will help people who want to try this themselves.

Reply:
done